cmis_web/static/src/cmis_folder/cmis_folder.js

```diff
@@ -157,6 +157,7 @@ export class CmisFolderField extends Component {
     }
 
     async createRootFolder() {
+        await this.props.record.save();
         if (!this.props.record.resId) {
```

Comment:
Shouldn't we call `save` after the if? If the record is not created, I don't know if `save()` won't raise an error?